### ISSUE TITLE
Does it work only in USA ? 

### ISSUE THREAD

**Issue by mastermind860** [gamagori/pizzapi#30]
I am from India. Can i use this for ordering pizza here ? if not, what should i do to make it work ?

**Comment by CoolMikey** [comment]
No, it only works in the USA and Canada, but I guess your question is does it work in India? - it doesn't work in India.
To be honest, I don't know if it is even possible, but my best guess is to modify the urls in `urls.py` to appropriate ones. Try adding a `COUNTRY_INDIA` in the `self.urls` variable and changing the `address.py` import at the beggining from: `from .urls import Urls, COUNTRY_USA` to: `from .urls import Urls, COUNTRY_INDIA` or whatever variable name You decide to use.
I would first try just to change the domain from `.com` to `.co.in`
If you succeed please send me a message. Good luck!

**Comment by gamagori** [comment]
@mastermind860 

Take a look at https://github.com/gamagori/pizzapi/blob/master/pizzapi/urls.py to get an idea for how the distinction between USA and Canada works.

It could be that the India API is entirely different, but I'd bet if you could figure out the URLs, you'd just have to insert a new entry in the urls.py file.

**Comment by iCrazyBlaze** [comment]
My fork supports UK, Japan and India. I don't know how well it works, but I added the regions. Try cloning it!
